api/Services/Validators/DurationValidator.js

```diff
@@ -0,0 +1,14 @@
+const DurationValidator = (duration) => {
+    duration = duration || 1
+    duration = parseFloat(duration)
+    if (typeof duration === 'number' && !isNaN(duration)) {
+        duration = Math.round(duration)
+        if (duration < 1)
+            duration = 1
+        return duration
+    } else {
+        return 1
+    }
```

Comment:
Since you're returning in the if statement, you can actually drop the else and just return 1 after the if